sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -593,9 +594,11 @@ def __init__(
     Args:
       task (str or enum.Enum): task supported by HuggingFace Pipelines.
         Accepts a string task or an enum.Enum from PipelineTask.
-      model : path to pretrained model on Hugging Face Models Hub to use custom
-        model for the chosen task. If the model already defines the task then
-        no need to specify the task parameter.
+      model : path to the pretrained model-id on Hugging Face Models Hub
+        to use custom model for the chosen task. If the model already defines
+        the task then no need to specify the task parameter. Use the model_id
+        string instead of an actual model here. Model specific kwargs can be
+        specified with model_kwargs using load_pipeline_args.
```

Comment:
it would help to format the text in docstings, for example take a look at huggingface docs in https://huggingface.co/docs/transformers/main_classes/model. Note how string constants are formatted italic (i think it's done via star-escaping (`*model id*`)) and code/arguments are escaped with back-ticks `. 

model (str): path to the pre-trained *model id* on Hugging Face Models Hub

Reply:
done